modular_skyrat/modules/delam_emergency_stop/code/scram.dm

```diff
@@ -211,55 +186,57 @@
 /obj/machinery/atmospherics/components/unary/delam_scram/proc/goodbye_friends()
-
 	// good job buddy, sacrificing yourself for the greater good
-	playsound(src, 'sound/machines/compiler/compiler-failure.ogg', 100, FALSE, MACHINE_SOUND_RANGE, ignore_walls = TRUE, use_reverb = TRUE, falloff_distance = MACHINE_SOUND_FALLOFF_DISTANCE)
+	playsound(
+		source = src,
+		soundin = 'sound/machines/compiler/compiler-failure.ogg',
+		vol = 100,
+		vary = FALSE,
+		extrarange = 15,
+		ignore_walls = TRUE,
+		use_reverb = TRUE,
+		falloff_distance = 10,
+	)
 	visible_message(span_danger("[src] beeps a sorrowful melody and collapses into a pile of twisted metal and foam!"), blind_message = span_danger("[src] beeps a sorrowful melody!"))
 	deconstruct(FALSE)
 
 /// Drain the internal energy, if the crystal damage is above 100 we heal it a bit. Not much, but should be good to let them recover.
-/obj/machinery/atmospherics/components/unary/delam_scram/proc/prevent_explosion()
+/obj/machinery/atmospherics/components/unary/delam_scram/proc/aftermath()
 	var/obj/machinery/power/supermatter_crystal/engine/damaged_sm = my_sm?.resolve()
 	if(!damaged_sm)
 		return
 
 	damaged_sm.name = "partially delaminated supermatter crystal"
 	damaged_sm.desc = "This crystal has seen better days, the glow seems off and the shards look brittle. Central says it's still \"relatively safe.\" They'd never lie to us, right?"
-	damaged_sm.explosion_power = SM_DAMAGED_EXPLOSION_POWER // if you fuck up again, yeesh
-
-	if(damaged_sm.damage > SM_PREVENT_EXPLOSION_THRESHOLD)
-		damaged_sm.damage = SM_PREVENT_EXPLOSION_THRESHOLD
+	damaged_sm.explosion_power += 5 // if you fuck up again, yeesh
+	for(var/current_gas_mix as anything in damaged_sm.current_gas_behavior)
+		var/datum/sm_gas/gas_mix = damaged_sm.current_gas_behavior[current_gas_mix]
+		if(istype(gas_mix, /datum/sm_gas/freon))
+			gas_mix.heat_resistance += 0.4
+			continue
+		gas_mix.heat_modifier += 1.5
+		gas_mix.heat_resistance -= 0.4
 
-	damaged_sm.internal_energy = MISTAKES_WERE_MADE
+	damaged_sm.internal_energy = 0
 	for(var/obj/machinery/power/energy_accumulator/tesla_coil/zappy_boi in range(3, src))
-		zappy_boi.stored_energy = MISTAKES_WERE_MADE
+		zappy_boi.stored_energy = 0
 
 /obj/machinery/atmospherics/components/unary/delam_scram/New()
 	. = ..()
 	var/datum/gas_mixture/delam_juice = new
 	delam_juice.add_gases(/datum/gas/freon)
-	delam_juice.gases[/datum/gas/freon][MOLES] = SM_COOLING_MIXTURE_MOLES
-	delam_juice.temperature = SM_COOLING_MIXTURE_TEMP
+	delam_juice.gases[/datum/gas/freon][MOLES] = 7000 // enough to stop most delams, but not the really big fuckups
+	delam_juice.temperature = 170 // 170K -103c
 	airs[1] = delam_juice
 
 /// A big red button you can smash to stop the supermatter engine, oh how tempting!
 /obj/machinery/button/delam_scram
-	name = "\improper supermatter emergency stop button"
+	name = "\proper the supermatter emergency stop button"
```

Comment:
whats up with this

Reply:
I want it to be called "the supermatter emergency stop button" instead of "a supermatter emergency stop button"